events/triggers/debug

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# function to upload the output to our paste server
+# Based on https://github.com/seejohnrun/haste-client#lightweight-alternative
+upload() { a=$(cat); curl -X POST -s -d "$a" https://umbrel-paste.vercel.app/documents | awk -F '"' '{print "https://umbrel-paste.vercel.app/"$6}'; }
```

Comment:
```suggestion

upload() {
  curl \
    --header "Content-Type: text/plain" \
    --request POST \
    --silent \
    --data-binary @- \
    https://debug.umbrel.tech/documents \
    | awk -F '"' '{print "https://debug.umbrel.tech/"$6}'
}
```

You can switch to the new URL here, fix the upload and make command more clear.

Reply:
Resolved in 1d131c5.